test/integration/webhook/workload_test.go

```diff
@@ -133,7 +133,7 @@ var _ = ginkgo.Describe("Workload validating webhook", func() {
 		},
 			ginkgo.Entry("podSets count less than 1", 0, 1, true),
 			ginkgo.Entry("podSets count more than 8", podSetsMaxItems+1, 1, true),
-			ginkgo.Entry("invalid podSet.Count", 3, 0, true),
+			ginkgo.Entry("valid podSet, count can be 0", 1, 0, false),
```

Comment:
```suggestion
			ginkgo.Entry("valid podSet, count can be 0", 3, 0, false),
```

Could you keep the previous case?

Reply:
Yes, sure